Tasks/XcodeV5/task.json

```diff
@@ -12,7 +12,7 @@
     "version": {
         "Major": 5,
         "Minor": 139,
```

Comment:
minor version should be 140 now, right?

Reply:
I'm planning to port this change back to releases/m139. Not that that means it has to stay 139.

Reply:
Ok -- I think it should stay 139 in that case